cf-deployment.yml

```diff
@@ -305,6 +305,7 @@ instance_groups:
             override: true
             refresh-token-validity: 2592000
             scope: cloud_controller.read,cloud_controller.write,openid,password.write,cloud_controller.admin,scim.read,scim.write,doppler.firehose,uaa.user,routing.router_groups.read,routing.router_groups.write
+            secret: ''
```

Comment:
Absolutely. We are moving away from defaults on passwords, private keys, etc. An empty string is still a deliberate value. So if a client is configured for this value, it has to be spelled out. 

This makes the manifest very clear on what values are required and what the actual value is. Even if operator decides to set the password of a client to `""`